tests/notificationConsumer.test.ts

```
@@ -0,0 +1,249 @@
+// --- mock dependencies ---
+jest.mock("../src/config/rabbitmq", () => ({
+  connectRabbitMQ: jest.fn(),
+}));
+
+jest.mock("../src/config/logger", () => ({
+  logger: { error: jest.fn(), info: jest.fn(), warn: jest.fn(), debug: jest.fn() },
```

ESLint

[error] Replace `·error:·jest.fn(),·info:·jest.fn(),·warn:·jest.fn(),·debug:·jest.fn()` with `⏎····error:·jest.fn(),⏎····info:·jest.fn(),⏎····warn:·jest.fn(),⏎····debug:·jest.fn(),⏎·`

(prettier/prettier)